docs/src/model_creation/dsl_advanced.md

```diff
@@ -272,6 +272,39 @@ sol = solve(oprob)
 plot(sol)
 ```
 
+### [Turning off species, parameter, and variable inferring](@id dsl_advanced_options_require_dec)
```

Comment:
"Turning off species, parameter, and variable **inference**"

sounds better to me (but I am not actually a native speaker, so defer actual judgment to you guys)